include/flatbuffers/idl.h

```diff
@@ -59,6 +59,21 @@ namespace flatbuffers {
   TD(STRUCT, "",       Offset<void>, int, int, int, int) \
   TD(UNION,  "",       Offset<void>, int, int, int, int)
 
+#define FLATBUFFERS_GEN_TYPES_SCALAR_ALIAS(TD) \
```

Comment:
I can do that, but the thinking is that whereever it is used would be more consistent with existing definitions and require less change if more aliases are updated. I'm happy either way.